erpnext/patches.txt

```diff
@@ -302,6 +302,8 @@ erpnext.patches.v13_0.custom_fields_for_taxjar_integration
 erpnext.patches.v14_0.delete_einvoicing_doctypes
 erpnext.patches.v13_0.set_operation_time_based_on_operating_cost
 erpnext.patches.v13_0.validate_options_for_data_field
+execute:frappe.delete_doc('Report', 'Sales Person Target Variance based on Item Group', ignore_missing=True)
+execute:frappe.delete_doc('Report', 'Sales Partner Target Variance based on Item Group', ignore_missing=True)
```

Comment:
Move this to a patch file, and don't delete them instead rename them with `merge=True`.

In case, if a user has set a shortcut to the report in their dashboard, then to preserve that link we will have to rename the old report name to the new one.